components/IconComponent.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts">
```

Comment:
Please add a comment here that this is a copy from another package (also mention the name of the package 🤗 ), since the code isn't ours. The license of the package doesn't require it, but it would be good for us in terms of reliability.